bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/config/TestBigtableVaneerSettingsFactory.java

```diff
@@ -0,0 +1,414 @@
+/*
```

Comment:
wrong spelling of vaneer in file name. I think it should be veneer

Reply:
Yupp Would change it to veneer, Thanks for pointing out.

Reply:
Shouldn't you change all vaneer to veneer in that case? 
BigtableVaneerSettingsFactory.createBigtableDataSettings(bigtableOptions); It appears in many places.

Reply:
I have renamed it in respective places, It appears in old history, If you go to [PR21/files](https://github.com/rahulKQL/cloud-bigtable-client/pull/21/files) you would not see any Vaneer reference.